content/setup/local-with-vagrant/_index.md

```diff
@@ -0,0 +1,264 @@
+
+    Login Succeeded
+    latest: Pulling from tinkerbell/tink-worker
+    Digest: sha256:55c99b7cfafd28244fd3f7adbe46d94ada305f0f28553f95dfd241c168f0439e
+    Status: Image is up to date for quay.io/tinkerbell/tink-worker:latest
+    quay.io/tinkerbell/tink-worker:latest
+    The push refers to repository [192.168.1.1/tink-worker]
+    c83ceae18bd4: Layer already exists
+    a1872780b5d4: Layer already exists
+    3e207b409db3: Layer already exists
+    latest: digest: sha256:55c99b7cfafd28244fd3f7adbe46d94ada305f0f28553f95dfd241c168f0439e size: 949
+    INFO: tinkerbell stack setup completed successfully on ubuntu server
+    NEXT: 1. Enter ./deploy and run: source ../envrc; docker-compose up
+           2. Try executing your fist workflow.
+              Follow the steps described in https://tinkerbell.org/examples/hello-world/ to say 'Hello World!' with a workflow.
+```
+
+When the provisioner is ready, a summary is printed to the console and you can now ssh in.
+
+### Connecting to the Provisioner
+
+All the following commands have to be executed in the artifacts' directory created above unless stated otherwise.
+
+Connect to the provisioner via ssh, vagrant has an until:
+
+```
+$ vagrant ssh provisioner
+vagrant@provisioner:~$
+```
+
+At this point, you are in an Ubuntu box that has a couple of utils installed and
+it is not time to start the services to make it a proper Tinkerbell provisioner
+
+```
+$ sudo su -l
+$ cd /vagrant && source envrc && cd deploy
+$ docker-compose up -d
+```
+
+Those commands give you a fully working provisioner that is ready to receive
+templates and workflow targeting workers. You can check that running:
+
+```
+$ docker-compose ps
+        Name                      Command                  State                             Ports
+-------------------------------------------------------------------------------------------------------------------------
+deploy_boots_1         /boots -dhcp-addr 0.0.0.0: ...   Up
+deploy_cacher_1        /cacher                          Up             0.0.0.0:42111->42111/tcp, 0.0.0.0:42112->42112/tcp
+deploy_db_1            docker-entrypoint.sh postgres    Up (healthy)   0.0.0.0:5432->5432/tcp
+deploy_hegel_1         cmd/hegel                        Up
+deploy_nginx_1         /docker-entrypoint.sh ngin ...   Up             192.168.1.2:80->80/tcp
+deploy_registry_1      /entrypoint.sh /etc/docker ...   Up (healthy)
+deploy_tink-cli_1      /bin/sh -c sleep infinity        Up
+deploy_tink-server_1   tink-server                      Up (healthy)   0.0.0.0:42113->42113/tcp, 0.0.0.0:42114->42114/tcp
+```
+
+### Register the worker's hardware
+
+The provisioner is running we can follow the [example called "Hello
+world"](/examples/hello-world), I will contextualize it for our vagrant setup
+here.
+
+SSH in the provisioner again and we have to define our hello world template and load it to Tinkerbell:
+
+```
```

Comment:
Do we have an official example of a real Operating System yet that we can link to?

Reply:
Not yet, no.

Reply:
Nop